### PR TITLE
deleted case for deprecated interface in getMock

### PR DESCRIPTION
**Pull request by ahmetakbn**
### Subject

Improving tests for `SonataCoreBundle`

### CODE REVIEW

**Comment by greg0ire** [comment]
When was `Symfony\Component\Validator\ExecutionContextInterface` removed ? Can you add this to your commit message ?

File: Tests/Validator/InlineValidatorTest.php [hunk 1]
```diff
@@ -31,11 +31,7 @@ public function setUp()
         $this->constraintValidatorFactory = $this->getMock(
             'Symfony\Component\Validator\ConstraintValidatorFactoryInterface'
         );
-        $this->context = $this->getMock(
-            interface_exists('Symfony\Component\Validator\Context\ExecutionContextInterface') ?
-                'Symfony\Component\Validator\Context\ExecutionContextInterface' :
```
Comment (Soullivaneuh) [inline Tests/Validator/InlineValidatorTest.php]: We support Symfony 2.3, we can't remove this.

Comment (greg0ire) [inline Tests/Validator/InlineValidatorTest.php]: Hence my question ;)

**Comment by greg0ire** [comment]
Close this PR if you agree @ahmetakbn

**Comment by Soullivaneuh** [comment]
Proof: https://travis-ci.org/sonata-project/SonataCoreBundle/jobs/137748230#L436

Closing this one.

**Comment by ahmetakbn** [comment]
Ok, everything is clear. I thought there is only support for Symfony 3 in versions 3.x. I think there is also support for Symfony 3 ;)

**Pull request closed without merging**